GenderPayGap.WebUI/Views/Viewing/Finder/Parts/Facets/Keywords.cshtml

```diff
@@ -28,22 +25,25 @@
         <div class="gov-uk-c-searchbar">
             <div class="gov-uk-l-searchbar__table">
                 <div class="gov-uk-l-searchbar__cell">
-                    <input id="FinderKeywords"
-                           value="@(Model.search)"
-                           aria-expanded="false"
-                           class="gov-uk-c-searchbar__input"
-                           name="search"
-                           type="text"
-                           autocomplete="off"
-                           spellcheck="false"
-                           maxlength="100"/>
+                    <label id="FinderKeywordsId" for="FinderKeywords" class="visuallyhidden">@(!string.IsNullOrEmpty(Model.t) && Model.t == "1" ? "Search by employer name" : "Search query")</label>
+                    <input class="gov-uk-c-searchbar__input" type="text" autocomplete="off" name="search" id="FinderKeywords">
                 </div>
                 <div class="gov-uk-l-searchbar__cell">
                     <button class="gov-uk-c-searchbar__submit gov-uk-c-searchbar__submit--blue" id="NextStep" type="submit">Search</button>
                 </div>
-
             </div>
         </div>
     </fieldset>
+</div>
 
-</div>
\ No newline at end of file
+<script>
+    function handleClick(radioValue) {
+        var element = document.getElementById("FinderKeywordsId");
+        if (radioValue == 1) {
+            element.textContent = "Search by employer name";
+        }
+        if (radioValue == 2) {
+            element.textContent = "Search by employer type or SIC code";
+        }
+    }
+</script>
```

Comment:
I'm not certain this is the right place for this code to live, but it does keep it close to where it is called. I did try and include it in the `@section Scripts` in `SearchResults.cshtml` from where this partial is called. But the function was out of scope.

Reply:
Can it live in the main js file for the old design?

Reply:
I think it's ok here.
I'd give it a more descriptive name though - perhaps `searchTypeRadioButtonClicked`

Reply:
Nice idea, I've changed that now